app/repository/document.py

```diff
@@ -133,6 +134,11 @@ def _doc_to_dto(doc_query_return: ReadObj) -> DocumentReadDTO:
         created=cast(datetime, fdoc.created),
         last_modified=cast(datetime, fdoc.last_modified),
         slug=str(fdoc.slugs[0].name if len(fdoc.slugs) > 0 else ""),
+        metadata=(
+            cast(dict, fdoc.valid_metadata)
+            if fdoc.valid_metadata is not None
+            else {"role": []}
```

Comment:
I wonder if we should pull out a `default_document_metadata` variable for this?

Reply:
Another thought - why do we need "role" on there? Why can't it be an empty dict? Or is this just temp while we change over?

Reply:
Can do. Worth noting that my next PR after I've tested this on staging will be to do an UPDATE on the documents table to create a JSON agg object to populate the metadata field if it's NULL by reading from the value in the role column  - so might be wasted effort

Reply:
yeah happy for you to leave as-is if that is the case